recipes-bsp/firmware-imx/firmware-imx-8.25.inc

```diff
@@ -1,16 +1,15 @@
 # Copyright (C) 2012-2016 Freescale Semiconductor
-# Copyright 2017-2023 NXP
+# Copyright 2017-2024 NXP
 # Copyright (C) 2018 O.S. Systems Software LTDA.
 SECTION = "base"
 LICENSE = "Proprietary"
-LIC_FILES_CHKSUM = "file://COPYING;md5=10c0fda810c63b052409b15a5445671a"
+LIC_FILES_CHKSUM = "file://COPYING;md5=ca53281cc0caa7e320d4945a896fb837"
 
 # Note: This .inc file is used from differently named recipes, so the package
 # name must be hard-coded, i.e., ${BPN} cannot be used.
 SRC_URI = "${FSL_MIRROR}/firmware-imx-${PV}-${IMX_SRCREV_ABBREV}.bin;fsl-eula=true"
-IMX_SRCREV_ABBREV = "fbe0a4c"
-SRC_URI[md5sum] = "440b125e897614d77fac663d90bcffc8"
-SRC_URI[sha256sum] = "2e27962332197ebebbb30138f6dfb365361d48d7efa565df97c4f79285b1ca50"
+IMX_SRCREV_ABBREV = "27879f8"
+SRC_URI[sha256sum] = "55766dad38961fde3ed3224ee6a46ab1c5aea62a5b947530228d8fbc4a7d2816"
```

Comment:
I believe the md5sum you removed is needed, have you checked? If I remember correctly the fsl-eula bbclass inside this layer will check the md5, which is not true for meta-imx.

Reply:
`SRC_URI` is not need for `fsl-eula-unpack.bbclass` (it will check EULA's md5sum only). `SRC_URI` need only for the general fetchers of bitbake. It is just optional to define md5sum and sha256sum together, only one of them important.

Reply:
Ok, thanks for clarifying, I remember a recipe breaking because we did not set the md5sum, that is why I was wondering if this is needed here. If not, all good, thanks!